css/block/video-reveal.css

```diff
@@ -56,7 +56,7 @@ span.ucb-vid-reveal-icon {
 /* Needed to overwrite the imageMedia Style that adds extra padding to images sitewide */
 .ucb-video-reveal-image > div > .contextual-region > .imageMediaStyle > img {
   padding-bottom: 0px;
-  max-height: 600px;
+  max-height: 575px;
```

Comment:
Did this just get caught in a search/replace for 600px or do we really need to adjust the height of the images on the video reveal based on the width changes?